Add button to toggle display of owned items

epinter opened the pull request.

epinter committed:
Add button to toggle display of owned items

beaverusiv commented:
I would prefer this setting in the filter items dialog instead of on the toolbar as it matches the purpose of that dialog (see: use checkboxes).

Auto refresh would be a timer calling `Files.reloadFiles(); refreshDisplay();`.

The other thing to account for is headings for categories that no longer have any non-found items. This is the main reason I didn't tackle it myself.

epinter committed:
Keep selected tabs after refresh

epinter committed:
Add a refresh timer to reload all files.

The time is controlled by RefreshTimer config. 0 is disabled (default), minimum 120 seconds.

epinter commented:
The timer is ready, see my fork ...
I don't see problem keeping the headings when category is empty, personally I prefer to keep the heading with the counter than see an emtpy tab content...
